### PR TITLE
MTA-2736 Added Welsh content for 400 / 404 / 500 error messages

### PR DESCRIPTION
**Pull request by richardjamesbaconHMRC**

### CODE REVIEW

File: src/main/resources/messages [hunk 1]
```diff
@@ -2,8 +2,8 @@ global.error.badRequest400.title=Bad request - 400
 global.error.badRequest400.heading=Bad request
 global.error.badRequest400.message=Please check that you have entered the correct web address.
 global.error.pageNotFound404.title=Page not found - 404
-global.error.pageNotFound404.heading=This page can’t be found
+global.error.pageNotFound404.heading=This page can''t be found
```
Comment (gitplaneta) [inline src/main/resources/messages:5]: Why did you use double quote?

File: src/main/resources/messages [hunk 2]
```diff
@@ -2,8 +2,8 @@ global.error.badRequest400.title=Bad request - 400
...
 global.error.pageNotFound404.message=Please check that you have entered the correct web address.
 global.error.InternalServerError500.title=Sorry, we are experiencing technical difficulties - 500
-global.error.InternalServerError500.heading=Sorry, we’re experiencing technical difficulties
-global.error.InternalServerError500.message=Please try again in a few minutes.
\ No newline at end of file
+global.error.InternalServerError500.heading=Sorry, we''re experiencing technical difficulties
```
Comment (gitplaneta) [inline src/main/resources/messages:8]: Same here, was this quote a mistake?

File: src/main/resources/messages (hunk 1 above)
Comment (richardjamesbaconHMRC) [reply src/main/resources/messages:5]: One of our previous requests for accessibility was to change primes (′) to apostrophes (') for screen readers. These are the corrections, but are escaped in the messages files to allow correct output

File: src/main/resources/messages (hunk 2 above)
Comment (richardjamesbaconHMRC) [reply src/main/resources/messages:8]: One of our previous requests for accessibility was to change primes (′) to apostrophes (') for screen readers. These are the corrections, but are escaped in the messages files to allow correct output

**Pull request merged**